packages/yoga/src/Checkbox/native/CheckMark.jsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { bool } from 'prop-types';
+import styled from 'styled-components';
+import { hexToRgb } from '@gympass/yoga-common';
+import check from './CheckIcon';
+
+const CheckMarkStyled = styled.View(
+  ({
+    checked,
+    disabled,
+    error,
+    theme: {
+      components: { checkbox },
+    },
+  }) => `
+    width: 24px;
+    height: 24px;
+
+    border-radius: ${checkbox.border.radii}px;
+    border-width: ${checkbox.border.width}px;
+    border-style: solid;
+
+    ${checked ? `background-color: ${checkbox.checked.backgroundColor};` : ''}
```

Comment:
In this case If the checkbox isn't checked the background will be transparent, is it correct?

Reply:
In this moment we will not use a background or inverted state, maybe in a second moment we can implement this behavior